com.ibm.streamsx.topology/opt/python/packages/streamsx/rest.py

```diff
@@ -24,31 +21,39 @@ class StreamsConnection:
     information.
 
     Example:
+        >>> # If the resource url does not have a valid SSL certificate, you can disable the SSL verification warning
+        >>> from requests.packages import urllib3
+        >>> urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+        >>>
         >>> _resource_url = "https://streamsqse.localdomain:8443/streams/rest/resources"
-        >>> sc = StreamsConnection(username="streamsadmin", password="passw0rd", resource_url=_resource_url)
+        >>> # By default SSL verification is enabled, set verify=False to disable it
+        >>> sc = StreamsConnection(username="streamsadmin", password="passw0rd", resource_url=_resource_url, verify=False)
```

Comment:
`verify` is an example of Request session object arguments that can be supplied to `session_kwargs`.